fixtures/multiple-paragraphs/expected.json

```diff
@@ -0,0 +1,3 @@
+{
+  "textarea": "1st paragraph\n\n2nd paragraph"
```

Comment:
Maybe worth adding another case for a single line break since it is not covered yet. 

```
1st paragraph\n2nd paragraph
```

If you don't mind, I'm happy to make the change on top of your changes and push it onto this branch.

Reply:
Here it is 5360a1d.

While updating the `id` from `textarea` to `textarea-one`, I stumbled across the replacing we do and I wonder if this is actually necessary. If I remember correctly, this was done to produce a nice looking output variable, but in the JSON output this is quite confusing. What do you think, should we get rid of this replace?

https://github.com/stefanbuck/github-issue-parser/blob/c4bc96debc0604cb0be0b9c3d5abd30747fdc01c/index.js#L47

Reply:
> If you don't mind, I'm happy to make the change on top of your changes and push it onto this branch.

no problem 👍🏼 

> While updating the `id` from `textarea` to `textarea-one`, I stumbled across the replacing we do and I wonder if this is actually necessary. If I remember correctly, this was done to produce a nice looking output variable, but in the JSON output this is quite confusing. What do you think, should we get rid of this replace?

Yeah +1 on getting rid of that. I think it will confuse people, I don't think this behavior is currently documented either. But maybe fix that in a  separate PR?

Reply:
Great, I will work on a pull request tomorrow evening. This is a breaking change so I'll update the workflow to push a `v2` branch JTLYK.